JSTOR-Elasticsearch-with-embeddings.py

```diff
@@ -0,0 +1,540 @@
+import logging
+import sys
+import re
+import xml.etree.ElementTree as ET
+import xmltodict
+
+from lxml import etree
+import os
+import traceback
+import elasticsearch
+import elasticsearch.helpers
+import json
+from enum import Enum
+from scipy.spatial.distance import cosine
+
+import torch
+from transformers import BertTokenizer, BertModel
+
+# CONFIGS
```

Comment:
Maybe all these would be better of as ENV variables that way they can be easily changed.